### DISCUSSION TITLE
SharePoint PnP Search Results Webpart -Results for a  site collection

### DISCUSSION THREAD

**Discussion by Rameez712** [microsoft-search/pnp-modern-search#4171] [Q&A] [answered]
Hi,

User added 'PnP Search Results' WebPart on Sitecollection however it is showing results from across the tenant.
I want it to search only 1 site collection

 I Tried
1. modiying the Query Template field to Path:{https://tenant.com/sites/sitecollectionName}
   No Changes happened
3. I also tried to select Result Source Id / Scope|Name to 'Local SharePoint Results'
    Results changed to 0

**Comment by kasperbolarsen** [comment, marked as answer]
you were sooo close:
Path:"https://tenant.com/sites/sitecollectionName*"

**Comment by Rameez712** [comment]
> you were sooo close: Path:"https://tenant.com/sites/sitecollectionName*"

Thanks @kasperbolarsen that worked....really appreciate
